### DISCUSSION TITLE
Documentation about requested scopes

### DISCUSSION THREAD

**Discussion by m1212e** [zitadel/zitadel#6191] [General]
Hi,

not sure where to put this but I wanted to give some feedback: First of all, ZITADEL is very nice, thanks!
But as a newbie to OpenID and ZITADEL I spent a lot of time the last week to find out why my tokens did not contain any user metadata at all. Then, randomly, I stumbled across request scopes and what they are. Maybe a small hint in the docs could save some people a lot of time. Maybe its even there and I overread it, but this did cost me days, so it would probably be very helpful, especially for beginners or folks who dont use one of your templates. 

Thanks again!

**Comment by hifabienne** [comment]
Hei @m1212e 
Thanks for your feedback.
Which guide did you follow?
